app/views/account.box.edit.location.html

```diff
@@ -15,14 +15,18 @@ <h2 style="margin-top: 0px; margin-bottom: 0px">{{'BOX_POSITION'|translate}}</h2
   <div style="height: 500px;">
     <osem-map id="map_edit" markers="location.editMarker" center=""></osem-map>
   </div>
-  <div ng-class="{'has-error': locationSettings.boxLat.$invalid, 'has-success': locationSettings.boxLat.$valid}" class="col-md-6">
+  <div ng-class="{'has-error': locationSettings.boxLat.$invalid, 'has-success': locationSettings.boxLat.$valid}" class="col-md-4">
     <label for="boxLat">{{'BOX_LATITUDE'|translate}}</label>
     <input name="boxLat" class="form-control" type="number" min="-85.06" max="85.06" ng-model="location.editMarkerInput.lat" step="0.000001" required>
   </div>
-  <div ng-class="{'has-error': locationSettings.boxLong.$invalid, 'has-success': locationSettings.boxLong.$valid}" class="col-md-6">
+  <div ng-class="{'has-error': locationSettings.boxLong.$invalid, 'has-success': locationSettings.boxLong.$valid}" class="col-md-4">
     <label for="boxLong">{{'BOX_LONGITUDE'|translate}}</label>
     <input name="boxLong" class="form-control" type="number" min="-180" max="180" ng-model="location.editMarkerInput.lng" step="0.000001" required>
   </div>
+  <div ng-class="{'has-error': locationSettings.boxHeight.$invalid, 'has-success': locationSettings.boxHeight.$valid}" class="col-md-4">
```

Comment:
Is the height ever set by the `locationfound` [event](http://leafletjs.com/reference-1.2.0.html#locationevent)?

@noerw Or should the user type in the altitude?

Reply:
I was not aware that the geolocation api can return altitude as well. will add this

Reply:
Maybe you can check if you can retrieve the altitude if the user changes the location

Reply:
Works now